### DISCUSSION TITLE
Worker has to be an other handbrake-container?

### DISCUSSION THREAD

**Discussion by dancgn** [TheNickOfTime/handbrake-web#150] [Q&A] [answered]
Hello there,

I installed it but can't get a worker running.

`     - WORKER_ID=myname
      - SERVER_URL=172.17.0.5 
      - SERVER_PORT=5808`

IP is the IP from Portainer-Container handbrake and the port. I've tried it with localhost and the IP of the system, nothing work.

Could you please make a tutorial for n00bs. Thanks

**Comment by TheNickOfTime** [comment, marked as answer]
Hey, please see #142
